server/views/pages/goal/review/index.njk

```diff
@@ -16,23 +16,24 @@
       <h1 class="govuk-heading-l">Review {{ prisonerSummary.firstName }} {{ prisonerSummary.lastName }}'s goals</h1>
 
       {% for goal in goals %}
+      {% set goalLoop = loop %}
```

Comment:
This is so we can track the `{% for goal in goals %}` index number for Change link URL nested within the `{% for step in goal.steps %}` loop (see above comment)